cmd/gateway/azure/gateway-azure.go

```diff
@@ -72,6 +72,11 @@ ENVIRONMENT VARIABLES:
   BROWSER:
      MINIO_BROWSER: To disable web browser access, set this value to "off".
 
+	CACHE:
+		 MINIO_CACHE_DRIVES: List of cache drives delimited by ";"
```

Comment:
this perhaps whole tabbing is off here you should align this with other entries here...